### PR TITLE
communication: the art of body language

### PR DESCRIPTION
**Pull request by jessyphung**

### CODE REVIEW

**Comment by monotykamary** [comment]
https://www.ted.com/talks/amy_cuddy_your_body_language_may_shape_who_you_are/transcript
http://www.permondo.eu/your-body-language-shapes-who-you-are/

There are quite a handful of phrases that are not unique in this note. Please see our contributing document on plagiarism: https://github.com/dwarvesf/brain/blob/master/CONTRIBUTING.md

This is probably just an honest mistake, but please keep in mind that we want the articles to be unique to the world view of the author. Please revise the note to paraphrase and recompose the idea as to how you might say or word it.

File: Communication/the art of body language.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+--
+tags: communication, language, psychology
+author: Phung Thi Thuong Thuong
+--
```
Comment (monotykamary) [inline Communication/the art of body language.md:4]: The metadata table should be wrapped with 3 hypens instead of 2.

**Review by monotykamary — CHANGES_REQUESTED** [review]

File: Communication/the art of body language.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+--
+tags: communication, language, psychology
+author: Phung Thi Thuong Thuong
+--
+# What is body language?
```
Comment (zlatanpham) [inline Communication/the art of body language.md:5]:
```suggestion
## What is body language?

```

File: Communication/the art of body language.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+--
+tags: communication, language, psychology
+author: Phung Thi Thuong Thuong
+--
+# What is body language?
+
+Body language is something that we're all fascinated with. Body language is nonverbal communication, and social scientists have spent a lot of time studying how our own body language, as well as the body language of others, affects our judgments. We form broad conclusions and generalizations based on body language. And such decisions may have a significant impact on our lives, such as who we employ or promote, or who we ask out on a date. For example, Nalini Ambady, a Tufts University researcher, found that when individuals view 30-second soundless video of genuine physician-patient encounters, their perceptions of the doctor's niceness predict whether or not the doctor would be sued. 
+
+# The impact of it
```
Comment (zlatanpham) [inline Communication/the art of body language.md:9]:
```suggestion
## The impact of it

```

File: Communication/the art of body language.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Body language is something that we're all fascinated with. Body language is nonverbal communication, and social scientists have spent a lot of time studying how our own body language, as well as the body language of others, affects our judgments. We form broad conclusions and generalizations based on body language. And such decisions may have a significant impact on our lives, such as who we employ or promote, or who we ask out on a date. For example, Nalini Ambady, a Tufts University researcher, found that when individuals view 30-second soundless video of genuine physician-patient encounters, their perceptions of the doctor's niceness predict whether or not the doctor would be sued. 
+
+# The impact of it
+Body language has a significant impact on how we assess people, how they judge us, and the results. Nonverbal signs of power and dominance in the animal kingdom include growing to appear larger and so more powerful, and humans do the same. When we feel helpless, on the other hand, we close down and shrink. Is it feasible, however, that we might persuade individuals to lie in order to encourage them to engage more and enhance their grades? 
+
+# How it works?
```
Comment (zlatanpham) [inline Communication/the art of body language.md:12]:
```suggestion
## How does it work?

```

File: Communication/the art of body language.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+You are more likely to feel strong if you act forceful. Our bodies can alter our brains, just as our minds may change our bodies. There are several distinctions between powerful and powerless individuals. There are also physiological variations in two crucial hormones: testosterone (the dominance hormone) and cortisol (the stress hormone). In a study, we discovered that people's hormones and stress levels vary depending on their posture.
+
+# Advices
```
Comment (zlatanpham) [inline Communication/the art of body language.md:15]:
```suggestion
## Advice 

```

**Review by zlatanpham — COMMENTED** [review]
Some nits

**Pull request merged**